### PR TITLE
Modify XLNet docstrings

### PR DESCRIPTION
**Pull request by yingyibiao**
### PR types

New features
### PR changes

Docs
### Describe

Add example code, modify doctoring format.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -26,25 +26,14 @@
...
-# sys.path.insert(0, os.path.abspath('../paddlenlp/models'))
-# sys.path.insert(0, os.path.abspath('../paddlenlp/seq2vec'))
-# sys.path.insert(0, os.path.abspath('../paddlenlp/transformers'))
-# sys.path.insert(0, os.path.abspath('../paddlenlp/utils'))
 
 # -- Project information -----------------------------------------------------
 
 project = 'PaddleNLP'
 copyright = '2021, nlpers'
 author = 'nlpers'
```
Comment (ZeyuChen) [inline docs/conf.py]: Author改成PaddleNLP

File: docs/conf.py [hunk 2]
```diff
@@ -26,25 +26,14 @@
...
-# sys.path.insert(0, os.path.abspath('../paddlenlp/metrics'))
-# sys.path.insert(0, os.path.abspath('../paddlenlp/models'))
-# sys.path.insert(0, os.path.abspath('../paddlenlp/seq2vec'))
-# sys.path.insert(0, os.path.abspath('../paddlenlp/transformers'))
-# sys.path.insert(0, os.path.abspath('../paddlenlp/utils'))
 
 # -- Project information -----------------------------------------------------
 
 project = 'PaddleNLP'
 copyright = '2021, nlpers'
```
Comment (ZeyuChen) [inline docs/conf.py]: CopyRight -> 2021, PaddleNLP

**Review by ZeyuChen — COMMENTED** [review]
Modify the author and copyright

File: docs/conf.py (hunk 1 above)
Comment (yingyibiao) [reply docs/conf.py]: done

File: docs/conf.py (hunk 2 above)
Comment (yingyibiao) [reply docs/conf.py]: done

**Review by ZeyuChen — APPROVED** [review]
LGTM

**Pull request merged**